opm/grid/cpgrid/Entity.hpp

```diff
@@ -316,8 +321,8 @@ namespace Dune
     } // namespace cpgrid
 } // namespace Dune
 
+#include <opm/grid/CpGrid.hpp>
```

Comment:
Thanks for noticing! It might have been there for a mistake when rebasing. Entity.hpp does not need CpGrid.hpp. 
It's removed in the last commit. Thanks!